src/rex.core/src/rex/core/testing.py

```diff
@@ -0,0 +1,91 @@
+#
+# Copyright (c) 2020, Prometheus Research, LLC
+#
+
+
+class RexTestSuite:
+    """
+    A ``pytest``-compatible test suite class that will automatically activate
+    a Rex application for the duration of the tests defined as part of the
+    class.
+
+    Requires that the ``create_app()`` method be implemented.
+    """
+
+    #: The currently-active Rex application.
+    rex = None
+
+    @classmethod
+    def create_app(cls):
+        """
+        Creates and returns the Rex application to activate for the tests
+        executed in this suite.
+
+        Must be implemented in concrete classes.
+
+        :rtype: rex.core.Rex
+        """
+
+        raise NotImplementedError(
+            f'{cls.__name__}.create_app() must be implemented'
+        )
+
+    @classmethod
+    def setup_class(cls):
+        cls.rex = cls.create_app()
+        cls.rex.on()
+
+    @classmethod
+    def teardown_class(cls):
+        cls.rex.off()
+        cls.rex = None
+
+
+class anything:
```

Comment:
Thinking more about this: probably `not_none` is useful in much more cases than `anything` cause `anything` is a really loose assertion (I don't remember I needed to check that key exists and don't do any assertion at least about its "nullability").

Still `anything` is useful when you don't really care about some part of the structure your asserting about.

Therefore I suggest:
* Rename `anything` => `ignored`
* Rename `not_none` => `anything_not_none`

Reply:
Renamed to `ignored` and `anything_except_none`.